src/loaders/configure/configure.js

```diff
@@ -14,8 +14,13 @@ export function configure (agentIdentifier, opts = {}, loaderType, forceDrain) {
     loader_config = nr.loader_config
   }
 
+  info.jsAttributes ??= {}
   if (isWorkerScope) { // add a default attr to all worker payloads
-    info.jsAttributes = { ...info.jsAttributes, isWorker: true }
+    info.jsAttributes.isWorker = true
+  }
+  if (isBrowserScope) { // retrieve & re-add all of the persisted setCustomAttribute|setUserId k-v from previous page load(s)
+    let prevPageSessionJsAttrs = getAllStorageItemsOfGroup(CUSTOM_ATTR_GROUP)
```

Comment:
We are basically saying here that `sessionStorage` custom attributes take precedence over passed-in info.jsAttributes definitions.  Is this expected?

Reply:
Wasn't on my mind, but now considering it, yes..? If someone wanted a custom attribute persisted, that implies it should have precedence and show up until such time they manually unset or change it.

It'll probably be one of divided opinion though. If I'm dev on team A working page A and I set it to persist, yes I'd want that value on refresh of page A over whatever my original info is. But if I'm dev on team B working on page B, and user navigates from A to B, I may not. So either,
- `persist` should only apply to the same page, rather than tab session, but this may defeat the perspective of tracking users thru multiple pages, OR
- team A takes the responsible of managing the unsetting themselves before user navigates to page B

Reply:
Yeah I think stick with behavior as currently defined for now until feedback is gathered